regression/pages/enterprise/enterprise_const.py

```diff
@@ -0,0 +1,22 @@
+"""
+URLs and constants for enterprise stuff
+"""
+
+import os
+
+
+DEFAULT_IDP_URL = u"https://idp.testshib.org/idp/Authn/UserPassword"
+
+IDP_URL = os.environ.get('IDP_URL', DEFAULT_IDP_URL)
+
+DEFAULT_IDP_NAME = "TestShib"
```

Comment:
Is this all based on an existing enterprise customer and tpa provider configuration? where does that configuration live?